Package.swift

```diff
@@ -1256,12 +1256,6 @@ let package = Package(
       exclude: [
         // Disable Swift tests as mixed targets are not supported (Xcode 12.3).
         "Unit/Swift",
-
-        // Disable Keychain dependent tests as they require a host application on iOS.
```

Comment:
Why add the requirement for a host application on iOS?

Reply:
The requirement of the host application is a SwiftPM limitation: https://forums.swift.org/t/host-application-for-spm-tests/24363/12

Reply:
I see, I read this backwards - as a new exclusion instead of an exclusion delete.